packages/e2fsprogs/e2fsprogs.spec

```diff
@@ -0,0 +1,125 @@
+Name: %{_cross_os}e2fsprogs
+Version: 1.45.6
+Release: 1%{?dist}
+Summary: Tools for managing ext2, ext3, and ext4 file systems
+License: GPL-2.0-only AND LGPL-2.0-only AND LGPL-2.0-or-later AND BSD-3-Clause
+URL: http://e2fsprogs.sourceforge.net/
+Source0: https://mirrors.edge.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/%{version}/e2fsprogs-%{version}.tar.xz
+Source10: mke2fs.conf
```

Comment:
```suggestion
# Include the distributed configuration in the repo so changes (which could cause incompatibility) are clearer
Source10: mke2fs.conf
```

Reply:
Added the comment to the config file instead, since it's less likely to be overlooked there.